lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Addresses.hs

```diff
@@ -224,3 +225,13 @@ spec = describe "SHELLEY_ADDRESSES" $ do
             (Link.listAddresses @'Shelley w) Default Empty
         expectResponseCode @IO HTTP.status404 r
         expectErrorMessage (errMsg404NoWallet $ w ^. walletId) r
+
+    it "ADDRESS_INSPECT_01 - Address inspect OK" $ \ctx -> do
+        let str = "Ae2tdPwUPEYz6ExfbWubiXPB6daUuhJxikMEb4eXRp5oKZBKZwrbJ2k7EZe"
+        r <- request @Aeson.Value ctx (Link.inspectAddress str) Default Empty
+        expectResponseCode @IO HTTP.status200 r
```

Comment:
any chance to also verify the result of this call here?

Reply:
Yes and no. It's technically possible of course. But I'll rather defer that to until we have a proper data-type for representing inspected data (as highlighted in a FIXME comment). But I'd rather introduce this data-type in cardano-addresses directly.